src/kursovaya/Main.java

```diff
@@ -0,0 +1,79 @@
+package kursovaya;
+
+public class Main {
+
+    public static Employee[] employees = new Employee[10];
+
+    static void getListEmployers() {
+        for (int i = 0; i < employees.length && employees[i] != null; i++) {
+            System.out.println(employees[i]);
+        }
+    }
+
+    static double sumSalaryMonth() {
+        double sum = 0;
+        for (int i = 0; i < employees.length && employees[i] != null; i++) {
```

Comment:
for (int i = 0; i < employees.length; i++) {
       if( employees[i] != null) {
            sum = sum + employees[i].getSalary();
        }
}